envs/python2/requirements.txt

```diff
@@ -1,2 +1,4 @@
-hiredis==0.2.0
-redis==2.10.5
+hiredis>=0.2.0
+redis>=2.10.5
+mockredis
+tabulate>=0.7.5
```

Comment:
This is a requirement for the status utility, but not for the actual circuit breaker module, so I added this to the requirements.txt files, but not to the setup.py